annotation/annotation/tasks/services.py

```diff
@@ -62,6 +62,34 @@
     ValidationSchema,
 )
 
+
+class LRU(OrderedDict):
+    def __init__(self, capacity):
+        super().__init__
+        self.capacity = capacity
+
+    def __contains__(self, item):
+        if super().__contains__(item):
+            self.move_to_end(item)
+            return True
+        else:
+            return False
```

Comment:
I dont think we need it. in operator just checks if item exists in the container, but it is not a retreival, so we shouldn't move it to end, meaning we dont need to redefine the method

Reply:
It was done because of the idea of cache, where any interaction with an element in cache memory would be flagged as "used". I can remove it certainly.